src/main/java/org/cloudfoundry/reconfiguration/tomee/GenericServicePropertiesProvider.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2016-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.reconfiguration.tomee;
+
+import org.apache.openejb.api.resource.PropertiesResourceProvider;
+import org.springframework.cloud.cloudfoundry.CloudFoundryRawServiceData;
+import org.springframework.cloud.cloudfoundry.com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.cloud.util.EnvironmentAccessor;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Created by tomcollings on 1/21/18.
+ */
+public class GenericServicePropertiesProvider implements PropertiesResourceProvider {
+
+    private ObjectMapper objectMapper;
+    private EnvironmentAccessor environment;
+
+    public GenericServicePropertiesProvider() {
+        objectMapper = new ObjectMapper();
+        environment = new EnvironmentAccessor();
+    }
+
+    /**
+     * The default configuration of the service provider with merged overrides. Set by OpenEJB <p> NOTE: The field name must be "properties" because that's the name expected by OpenEJB </p>
+     */
+    private Properties properties;
```

Comment:
Keep some reasonable javadoc row length

Reply:
Project margin is at 200 characters, so just ensure that you've used IDEA to format the project and that's what we'll enforce.